lstchain/scripts/lstchain_mc_r0_to_dl1.py

```diff
@@ -42,7 +43,11 @@
                     help='Path to a configuration file. If none is given, a standard configuration is applied',
                     default=None
                     )
-
+parser.add_argument('--dynamic-cleaning', action='store',
+                    type=lambda x: bool(strtobool(x)),
+                    dest='dynamic_cleaning',
+                    help='Boolean. True to use dynamic cleaning',
```

Comment:
Replace with "If passed, dynamic cleaning is used".

Reply:
Waaait... I was just trying if I could push code properly :'-D